.github/workflows/luacheck.yml

```diff
@@ -0,0 +1,17 @@
+name: luacheck
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
I think it's best to use `v2` or even `master`.
This one is old and is slower than `v2`.

Reply:
@Panquesito7 i've switched to `v2` for the actions/checkout image :+1: 
**Just fyi**: i'm thinking about creating a dedicated action for luachecks like this, the `apt-get` and `luarocks` are executed every time this gets invoked, not best in regards for resource-usage but it works for all my other projects :shrug: (and yes: i know the "factorio" luacheck action but i don't like the hardcoded absolute URL for the `.luacheckrc` file)

Reply:
You can use Factorio LuaCheck Action without specifying a `.luacheckrc` URL:

              luacheckrc_url: \"\"

See Roang-zero1/factorio-mod-luacheck#4 for more information. 🙂